openstack/datadog_checks/openstack/openstack.py

```diff
@@ -767,6 +775,8 @@ def ensure_auth_scope(self, instance):
         """
         instance_scope = None
 
+        custom_tags = instance.get('tags', [])
```

Comment:
should we check if that's `None`?

Reply:
yep! I'll add the check for tags being `None`.